packages/api/webpack.config.js

```diff
@@ -1,20 +1,40 @@
+import dotenv from 'dotenv'
 import { createRequire } from 'module'
+import SentryWebpackPlugin from '@sentry/webpack-plugin'
+import { GitRevisionPlugin } from 'git-revision-webpack-plugin'
```

Comment:
Needs to be added to dependencies?

Reply:
yes, I was playing it other options before PRing and removed, but did not put back